packages/amplication-git-utils/src/IGitProvider.ts

```diff
@@ -0,0 +1,46 @@
+import {
+  Branch,
+  CreateBranchIfNotExistsArgs,
+  CreateCommitArgs,
+  CreatePullRequestForBranchArgs,
+  CreatePullRequestFromFilesArgs,
+  CreateRepositoryArgs,
+  GetFileArgs,
+  GetPullRequestForBranchArgs,
+  GetRepositoriesArgs,
+  GetRepositoryArgs,
+  GitFile,
+  RemoteGitOrganization,
+  RemoteGitRepos,
+  RemoteGitRepository,
+} from "./types";
+
+export interface IGitProvider {
```

Comment:
There is no convention to prefix interfaces in Ts with `I` and we are not currently using this convention elsewhere. Please rename it to `GitProvider`